index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Odin Landing Page</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">Header Logo</div>
+      <div>header link one</div>
+      <div>header link two</div>
+      <div>header link three</div>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-text">
+          <h1>This website is awesome</h1>
+          <p>
+            This website has some subtext that goes here under the title. It's
+            smaller font and the color is lower contrast.
+          </p>
+          <button class="intro-button">Sign up</button>
+        </div>
+        <div class="intro-image"><p>this is a placeholder for an image</p></div>
+      </div>
+      <div class="info">
+        <h2>Some random information.</h2>
+        <ul>
+          <li>
+            <div class="info-box"></div>
+            <p>this is some subtext under an illustration or image</p>
+          </li>
+          <li>
+            <div class="info-box"></div>
+            <p>this is some subtext under an illustration or image</p>
+          </li>
+          <li>
+            <div class="info-box"></div>
+            <p>this is some subtext under an illustration or image</p>
+          </li>
+          <li>
+            <div class="info-box"></div>
+            <p>this is some subtext under an illustration or image</p>
+          </li>
+        </ul>
+      </div>
+      <div class="quote">
+        <p>
+          This is an inspiriong quote, or a testimonial from a customer. Maybe
+          it's just filling up space, or maybe people will actually read it. Who
+          knows? All I know is that it looks nice.
+        </p>
+        <p class="second-paragraph">-Thor, God of Thunder</p>
+      </div>
+      <div class="signup-box">
+        <div class="signup-text">
+          <h3>Call to action! It's time!</h3>
+          <p>
+            Sign up for our product by clicking that button right over there!
+          </p>
+        </div>
+        <button class="signup-button">Sign up</button>
+      </div>
+    </main>
+    <footer>Copyright The Odin Project 2021</footer>
+  </body>
+</html>
```

Comment:
add new line below, for more context you can refere this -- https://github.com/sahnavin123/recipes-project/pull/1#discussion_r1180268647